.gitignore

```diff
@@ -18,7 +18,7 @@ crash.*.log
 # password, private keys, and other secrets. These should not be part of version
 # control as they are data points which are potentially sensitive and subject
 # to change depending on the environment.
-*.tfvars
+
 *.tfvars.json
```

Comment:
i guess we nee {filename}.tfvars for keeping consistent code ?

Reply:
The .tfvars must not be ignored because contains the variables. I suggest to remove the row *.tfvars.json as well (not included in terraform gitignore, normally).

Reply:
Please approve my pull request ! 🍪